public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -279,8 +279,11 @@ const Policy: FC<PolicyComponentProps> = ({
                     <MuteTimings timings={muteTimings} alertManagerSourceName={alertManagerSourceName} />
                   </MetaText>
                 )}
-                {timingOptions && Object.values(timingOptions).some(Boolean) && (
-                  <TimingOptionsMeta timingOptions={timingOptions} />
+                {timingOptions && (
+                  // for the default policy we will also merge the default timings, that way a user can observe what the timing options would be
+                  <TimingOptionsMeta
+                    timingOptions={isDefaultPolicy ? { ...timingOptions, ...TIMING_OPTIONS_DEFAULTS } : timingOptions}
```

Comment:
I'm creating a new object inline here because I wanted to avoid `timingOptions` with applied defaults being accidentally passed in to a component that might make mutations to the policy. 

That's not happening in this file but I also want to prevent future regressions of this sort.